setup.py

```diff
@@ -162,6 +162,8 @@ def is_excluded_dir(dir):
         install_requires=REQUIRES,
         tests_require=TESTS_REQUIRE,
         extras_require=EXTRAS_REQUIRE,
-        packages=find_packages(exclude=EXCLUDE_PATTERNS),
+        packages=find_packages(
+            exclude=["workbench*", "shoop_tests*"] + EXCLUDE_PATTERNS
+        ),
```

Comment:
I don't like the idea of polluting client venvs with them.

Do we have some envvar(s) we could test against and include these when running tests?